clients/drcachesim/reader/record_file_reader.h

```diff
@@ -149,7 +149,13 @@ class record_reader_t : public std::iterator<std::input_iterator_tag, trace_entr
         UNUSED(res);
```

Comment:
> The key change here is having record_reader_t increment the instruction ordinal on an encoding record instead of an instruction record.

This should be documented in  get_instruction_ordinal in scheduler.h.

Reply:
Done.

Reply:
Also fixed the scheduler output stream counting.